Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM golang:1.13.4 AS build
+FROM golang:1.16 AS build
```

Comment:
Please also update the README that Go v1.16 (or later) is required

Reply:
Done 👍